src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.cap {
+  position: relative;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: inline-block;
+  align-items: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.box {
+  margin-right: 20px;
+  height: 60px;
+  line-height: 60px;
+  position: relative;
+}
+
+.last-box {
```

Comment:
use pseudo class for selecting the last element instead of setting separate class for it